acquire/acquire.py

```diff
@@ -1733,6 +1733,9 @@ def main():
         target = Target.open(target_path)
         log.info(target)
     except Exception:
+        if not is_user_admin():
+            log.error("Failed to load target, try re-running as administrator/root.")
+            exit(1)
```

Comment:
```suggestion
            parser.exit(1)
```
I don't think it actually matters, just for consistency sake because it's what's used a couple lines above.